transcription/transcription.go

```diff
@@ -29,3 +32,33 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// DownloadFileFromURL downloads an mp3 file locally from a url.
+func DownloadFileFromURL(url string) error {
+	// https://github.com/thbar/golang-playground/blob/master/download-files.go
+	tokens := strings.Split(url, "/")
+	fileName := tokens[len(tokens)-1]
+
+	// TODO: check file existence first with io.IsExist
```

Comment:
Yoni and I talked about this and we both agree that we don't need to do this - in what scenario would you even have two files that have the same name? What do you think @sandlerben - if so let's remove this comment